ui/src/components/DenylistTable/index.jsx

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { string, shape, func, arrayOf } from 'prop-types';
+import classNames from 'classnames';
+import { pipe, map, sort as rSort } from 'ramda';
+import memoize from 'fast-memoize';
+import { withStyles } from '@material-ui/core/styles';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import Typography from '@material-ui/core/Typography';
+import { notificationAddress, pageInfo } from '../../utils/prop-types';
+import { VIEW_DENYLIST_PAGE_SIZE } from '../../utils/constants';
+import sort from '../../utils/sort';
+import ConnectionDataTable from '../ConnectionDataTable';
+
+const sorted = pipe(
+  rSort((a, b) => sort(a.node.notificationAddress, b.node.notificationAddress)),
+  map(({ node: { notificationAddress } }) => notificationAddress)
+);
+const tableHeaders = ['Address', 'Type'];
+
+@withStyles(theme => ({
+  tableCell: {
+    textDecoration: 'none',
+  },
+  listItemCell: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '100%',
+    padding: theme.spacing.unit,
+  },
+  listLinkCell: {
+    ...theme.mixins.hover,
+  },
+}))
+export default class DenylistTable extends Component {
```

Comment:
The first column is missing the link icon:

![Screen Shot 2019-03-19 at 5 51 15 PM](https://user-images.githubusercontent.com/3766511/54644954-0bb9da00-4a71-11e9-851a-b870aaf21b52.png)